src/parser.js

```diff
@@ -5,7 +5,7 @@ const fs = require('fs');
 const { transform } = require('./transform');
 
 // this function is only for dev and can be removed for production
-const createAST = (filePath) => {
+const createAST = filePath => {
```

Comment:
Seems like we might be using different linting rules! We should touch base to pick which way we'd like to go with this one (parentheses or not around a single parameter in an arrow function like this.